.github/workflows/release-pr.yml

```diff
@@ -0,0 +1,72 @@
+name: Release PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Version to release"
+        required: true
+        default: "0.0.0"
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  release:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Branch
+        id: branch
+        run: |
+          version=${{ github.event.inputs.version }}
+          version=${version%.*}
+          echo "branch=release-$version" >> $GITHUB_OUTPUT
+
+      - name: Checkout Repository
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          ref: ${{ steps.branch.outputs.branch }}
+
+      - name: Vars
+        id: vars
+        run: |
+          echo "current_version=$(yq '.version' <deploy/helm-chart/Chart.yaml)" >> $GITHUB_OUTPUT
```

Comment:
the problem is that `appVersion` is `edge` in `main`. I think we should change it to the last released version, but I didn't want to make that change in this PR

Reply:
We'll have to see if there are any side effects of that change. It could cause confusion because we want the helm chart in main to install edge, but if a version is specified, that's not clear. And if we use appVersion for anything, that also messes things up.

Reply:
Could we run a `git` command to get the last tag?

Reply:
Helm says that `appVersion` is "The version of the app that this contains", so `1.1.0` doesn't contain `edge`.

But I also don't think putting `edge` there has any effect when you install the chart?

Reply:
As of right now, no, appVersion isn't used for anything programmatic. I wonder if people can use it as a form of documentation though. What I mean is that if they want to install edge using helm, they install from our main branch, but the appVersion says that it's `1.1.0`, even though it isn't. So it's a bit confusing.

Reply:
I meant more from a helm chart point of view, I think this file is only used for publishing, it doesn't really matter what we put here for the user, and technically if we have `version` `1.1.0` it should be `appVersion` `1.1.0` because that's what it was published.

yeah we could use `git`...it would be a much shorter command if we would tag in main instead of release branch 😬

Reply:
Hm, yeah good point about the version. We also do publish helm charts with edge values for the versions when we merge to main....so I'm changing my mind.

Reply:
so I should update the `appVersion` and use that? Is that what you mean?

Reply:
Yep